src/plugins/default/markbind-plugin-footnotes-popovers.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       popoversHtml += `
         <popover id="${id}">
           <div slot="content">
-            <trigger for="${id}">
+            <trigger for="${id}" add-class="footnote-item-popover">
```

Comment:
The trigger is to make it hoverable

Reply:
Ah i see the issue.. I'm abit hesitant with the change due to the `important` flag and it introduces a new class for this. Is there a way to edit `Popover.vue` or [`popoverMixin.js`](https://github.com/MarkBind/vue-strap/blob/2ab61c21313ed11d46646376766f82a7ebc42181/src/utils/popoverMixins.js) instead? I'm thinking of providing an option for the style/functionality.

I would imagine that it would be useful for popovers as well.

Reply:
Made the trigger component more customisable, removing the need for !important. https://github.com/MarkBind/vue-strap/pull/112

Reply:
> The trigger is to make it hoverable

Unrelated to this PR, but that is a very odd implementation. :P Was it not possible to fix the popover itself to make it hoverable, rather than having to use a trigger for it?

Reply:
I'll try doing that instead.

Reply:
Ok, https://github.com/MarkBind/vue-strap/pull/114 is merged, can you see if we can use that as an alternative way of fixing the underline?